packages/ketcher-react/src/script/editor/utils/selectItems/finishSelecting.ts

```diff
@@ -0,0 +1,92 @@
+import { Action, fromItemsFuse, FunctionalGroup, SGroup } from 'ketcher-core'
```

Comment:
feature related changes in this file -> mouseup logic based on `select` instrument without lasso